### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MathijsBols**
Added documentation for the icalander token.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1338,6 +1339,30 @@ You can either provide a date range or a single date. If you provide a single da
 
 </details>
 
+### ICalander: `GET /rest/v1/icalendar`
```
Comment (MattterSteege) [inline README.md:1342]:
```suggestion
### ICalendar: `GET /rest/v1/icalendar`
```

File: README.md [hunk 2]
```diff
@@ -1338,6 +1339,30 @@ You can either provide a date range or a single date. If you provide a single da
...
+<details><summary>Click to open</summary>
+
+Fetches the icalander token.
```
Comment (MattterSteege) [inline README.md:1345]:
```suggestion
Fetches the url to the icalendar stream.
```

File: README.md [hunk 3]
```diff
@@ -1338,6 +1339,30 @@ You can either provide a date range or a single date. If you provide a single da
...
+{
+    "links": [],
+    "permissions": [],
+    "additionalObjects": {},
+    "leerlingICalendarLink": "https://api.somtoday.nl/rest/v1/icalendar/stream/REDACTED"
+}
+```
+
+</details>
+
```
Comment (MattterSteege) [inline README.md:1365]:
```suggestion

### ICalendar: `DELETE /rest/v1/icalendar`
<details><summary>Click to open</summary>

Deletes the currently active icalendar stream

#### Parameters

| Name          | Type      | Value                 |
|---------------|-----------|-----------------------|
| Authorization | Header    | Bearer [access_token] |

#### Returns

NONE

</details>
```

File: README.md [hunk 4]
```diff
@@ -30,6 +30,7 @@
     - [School Years: `GET /rest/v1/schooljaren`](#schooljaren-get-restv1schooljaren--get-restv1schooljarenid)
     - [Vakkeuzes: `GET /rest/v1/vakkeuzes`](#vakkeuzes-get-restv1vakkeuzes)
     - [Waarnemingen: `GET /rest/v1/waarnemingen`](#waarnemingen-get-restv1waarnemingen)
+    - [ICallander: `GET /rest/v1/icalendar`](#icalander-get-restv1icalendar)
```
Comment (MattterSteege) [inline README.md:33]:
```suggestion
    - [ICalendar: `GET /rest/v1/icalendar`](#icalendar-get-restv1icalendar)
    - [ICalendar: `DELETE /rest/v1/icalendar`](#icalendar-delete-restv1icalendar)
    ```

**Review by MattterSteege — CHANGES_REQUESTED** [review]
Looking very nice, please look over the suggested changes to spelling and the addition of the `DELETE` request.

**Comment by MathijsBols** [comment]
Is it done now?

File: README.md [hunk 5]
```diff
@@ -30,6 +30,9 @@
     - [School Years: `GET /rest/v1/schooljaren`](#schooljaren-get-restv1schooljaren--get-restv1schooljarenid)
     - [Vakkeuzes: `GET /rest/v1/vakkeuzes`](#vakkeuzes-get-restv1vakkeuzes)
     - [Waarnemingen: `GET /rest/v1/waarnemingen`](#waarnemingen-get-restv1waarnemingen)
+    - [ICalendar: `GET /rest/v1/icalendar`](#icalendar-get-restv1icalendar)
+    - [ICalendar: `DELETE /rest/v1/icalendar`](#icalendar-delete-restv1icalendar)
+    ```
```
Comment (MattterSteege) [inline README.md:35]:
```suggestion
```

**Review by MattterSteege — CHANGES_REQUESTED** [review]

**Pull request merged**